consensus/src/block_storage/sync_manager.rs

```diff
@@ -68,6 +68,13 @@ impl BlockStore {
             && !self.block_exists(li.commit_info().id()))
             || self.commit_root().round() + 30.max(2 * self.vote_back_pressure_limit)
                 < li.commit_info().round()
+            // If the LI commit block timestamp is more than 60 secs ahead of self commit block 
+            // timestamp, sync to the ledger info
+            || li
+                .commit_info()
+                .timestamp_usecs()
+                .saturating_sub(self.commit_root().timestamp_usecs())
+                >= Duration::from_secs(60).as_micros()
```

Comment:
think more about it, I'd like to have this number smaller than the one below for more buffer as well, maybe 30s here?

Reply:
Hmm, there is still a 60s grace period to sync to higher LI and still have the payload available right?